config.js

```diff
@@ -1,7 +1,7 @@
 
 const RESOURCE_BASE = 'http://mu.semte.ch/services/file-bundling-service';
 const MU_APPLICATION_FILE_STORAGE_PATH = '';
-const STORAGE_PATH = `/share/${MU_APPLICATION_FILE_STORAGE_PATH}`;
+const STORAGE_PATH = `/archive/${MU_APPLICATION_FILE_STORAGE_PATH}`;
```

Comment:
Nothing technically breaks by doing this, it's convention and good practice to always use `/share` as the place where a semantic.works microservice puts/reads files. All other services (should) behave like this, so it's better to have the same behaviour everywhere.